source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -215,10 +218,15 @@ def expand(self,unit):
 				self.setEndPoint(docInfo,"endToEnd")
 
 	def getTextWithFields(self,formatConfig=None):
-		if self.isCollapsed:
-			# #7652: We cannot fetch fields on collapsed ranges otherwise we end up with repeating controlFields in braille (such as list list list). 
-			return []
-		fields=super(WordDocumentTextInfo,self).getTextWithFields(formatConfig=formatConfig)
+		fields = None
+		if not self.isCollapsed:
+			rawText = self._rangeObj.GetText(2)
+			if not rawText or rawText == END_OF_ROW_MARK:
+				r = self.copy()
+				r.end = r.start
+				fields = super(WordDocumentTextInfo, r).getTextWithFields(formatConfig=formatConfig)
+		if fields is None:
+			fields = super(WordDocumentTextInfo, self).getTextWithFields(formatConfig=formatConfig)

```

Comment:
Might as well make this new style super. That also creates a bigger contrast with the super call abbove, that calls super on r.

```suggestion
			fields = super().getTextWithFields(formatConfig=formatConfig)
```